src/overlays/actors/ovl_En_Talk_Gibud/z_en_talk_gibud.c

```diff
@@ -524,9 +524,9 @@ void EnTalkGibud_SetupStunned(EnTalkGibud* this) {
     this->actor.world.rot.y = this->actor.shape.rot.y;
 
     if (this->drawDmgEffTimer != 0) {
-        Actor_SetColorFilter(&this->actor, 0, 0xC8, 0, 0x28);
+        Actor_SetColorFilter(&this->actor, COLORFILTER_COLORFLAG_BLUE, 200, COLORFILTER_BUFFLAG_OPA, 0x28);
```

Comment:
```suggestion
        Actor_SetColorFilter(&this->actor, COLORFILTER_COLORFLAG_BLUE, 200, COLORFILTER_BUFFLAG_OPA, 40);
```
Same for the one below.